src/python/WMCore/WMStats/Service/ActiveRequestJobInfo.py

```diff
@@ -26,4 +26,39 @@ def validate(self, apiobj, method, api, param, safe):
     def get(self):
         # This assumes DataCahe is periodically updated. 
         # If data is not updated, need to check, dataCacheUpdate log
-        return rows([DataCache.getlatestJobData()])
\ No newline at end of file
+        return rows([DataCache.getlatestJobData()])
+    
+class ProtectedLFNList(RESTEntity):
```

Comment:
Caching is not only for this call. The cache is wmstats monitor page, since it is already there, we are just reusing it. So we have minimal performance penalty. :-)